app/controllers/application_controller.rb

```diff
@@ -9,5 +9,14 @@ def current_user
   	User.find(session[:user_id]) if session[:user_id]
   end
 
-  helper_method :current_user
+  def average_rating(obj)
```

Comment:
you can move this to the `video.rb` and do:
```ruby
reviews.average(:rating).round(1) if reviews.average(:rating)
```

What you have is not bad but when your app grows and you have thousands of reviews for a video `obj.reviews.each` will take quite some time.